### PR TITLE
Add network policies

### PR DESCRIPTION
**Pull request by jpapejr**
Adding sample Kubernetes Network Policy examples to compliment our Calico examples

### CODE REVIEW

File: calico-policies/private-network-isolation/calico-v3/generic-privatehostendpoint.yaml [hunk 1]
```diff
@@ -25,13 +25,254 @@ metadata:
...
+---
+
+# worker node 3
+apiVersion: projectcalico.org/v3
+kind: HostEndpoint
+metadata:
+  labels:
+    ibm.role: worker_private
+  # Run 'calicoctl get nodes' to see Calico-formatted worker node names
+  name: kube-dal10-cr5bd37250fa4b4961af57cbfc43774d59-w2.cloud.ibm-worker-private
```
Comment (jgarcows) [inline calico-policies/private-network-isolation/calico-v3/generic-privatehostendpoint.yaml]: <wnn>

**Comment by jpapejr** [comment]
@jgarcows oops.. that file wasn't supposed to be changed/committed.. I'll amend.. thx.

**Comment by jpapejr** [comment]
7e0a23467b1b7ebd6771e005a00b74bb6f617598 fixes the file mentioned.

**Comment by jgarcows** [comment]
LGTM

**Pull request merged**